packages/@aws-cdk/aws-servicecatalog/lib/product.ts

```diff
@@ -0,0 +1,221 @@
+import { IResource, Resource, Stack } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { AcceptLanguage } from './common';
+import { InputValidator } from './private/validation';
+import { CfnCloudFormationProduct } from './servicecatalog.generated';
+import { Template } from './template';
+
+/**
+ * A Service Catalog product, currently only supports type CloudFormationProduct
+ */
+export interface IProduct extends IResource {
+  /**
+   * The ARN of the product.
+   * @attribute
+   */
+  readonly productArn: string;
+
+  /**
+   * The id of the product
+   * @attribute
+   */
+  readonly productId: string;
+}
+
+/**
+ * A reference to a Service Catalog product.
+ */
+export interface ProductAttributes {
+  /**
+   * The ARN of the product.
+   */
+  readonly productArn: string;
+
+  /**
+   * The name of the product.
+   */
+  readonly productName: string;
+}
+
+abstract class ProductBase extends Resource implements IProduct {
+  public abstract readonly productArn: string;
+  public abstract readonly productId: string;
+}
+
+/**
+ * Properties of provisioning artifacts
+ */
+export interface ProvisioningArtifactProperties {
+  /**
+   * The description of the provisioning artifact
+   * @default - No description provided
+   */
+  readonly description?: string;
+
+  /**
+   * Disable template validation.
+   * @default True
```

Comment:
Isn't this an awful default? Does this mean, when I create a Production version with an invalid template, that I will only find out about it when my customer tries to use it?

Can we make the default `false` please?

Reply:
Makes sense, but since we are changing it to `validateTemplate` instead, it makes sense to leave the default at `true` right?

Reply:
Yep.